### PR TITLE
add feature flag to enable / disable conference timer

### PR DESCRIPTION
**Pull request by nicolasd**
This PR will add a "conference-timer.enabled" flag that will allow to show / hide the conference timer

### CODE REVIEW

**Comment by nicolasd** [comment]
done for my previous PR

**Comment by luixxiul** [comment]
Adding CLA-signed label based on https://github.com/jitsi/jitsi-meet/pull/6436#event-3332338299

File: react/features/conference/components/native/NavigationBar.js [hunk 1]
```diff
@@ -73,7 +78,9 @@ class NavigationBar extends Component<Props> {
                             { this.props._meetingName }
                         </Text>
                     }
-                    <ConferenceTimer />
+                    {
+                        this.props._conferenceTimerEnabled ? <ConferenceTimer /> : null
```
Comment (saghul) [inline react/features/conference/components/native/NavigationBar.js:82]:
```suggestion
                        this.props._conferenceTimerEnabled && <ConferenceTimer />
```

**Comment by nicolasd** [comment]
Thanks, i pushed a fix

**Comment by nicolasd** [comment]
@saghul would you have the time to merge this feature ? we would like to have this one on the next android sdk release

thank you!

**Comment by nicolasd** [comment]
@saghul any news about merging this feature? 

thanks

**Comment by saghul** [comment]
Apologies for the delay. Alas this missed the 2.9.1 cut, which happened yesterday. Will be part of the next release.

**Pull request merged**